src/main.ts

```diff
@@ -195,6 +196,31 @@ export default class ShikiPlugin extends Plugin {
 		}
 	}
 
+	registerInlineCodeProcessor(): void {
+		this.registerMarkdownPostProcessor(async (el, ctx) => {
+		    const inlineCodes = el.findAll(":not(pre) > code");
+		    for (let codeElm of inlineCodes) {
+		        let match = codeElm.textContent.match(/^\{([^\s]+)\} (.*)/i);  // format: `{lang} code`
```

Comment:
Since this regex appears twice, it would be best to put that into a constant somewhere and refer to that constant here and in the view plugin.

Reply:
I considered this but didn't know where to put the constant. At last I add it in main.ts.

```js
export const SHIKI_INLINE_REGEX = /^\{([^\s]+)\} (.*)/i;  // format: `{lang} code`
```